db/seed_helper.rb

```diff
@@ -5,4 +5,15 @@ def create_roles_and_permissions
   Role.create!(:name => Role::DATA_OWNER_ROLE)
 end
 
-
+# Populates the languages table with the language names and codes from the languages CSV file
+def populate_languages
+  Language.delete_all
```

Comment:
If you have an association between one model and a Language, and you delete all languages when the application is redeployed, then you have effectively broken that link.
You can use this: http://apidock.com/rails/v4.0.2/ActiveRecord/Relation/find_or_create_by